po/common/fr.po

```diff
@@ -200,8 +200,8 @@ msgid "All missions"
 msgstr "Toutes les missions"
 
 msgctxt "allergens"
-msgid "Substances or products causing allergies or intolerances"
-msgstr "Substances ou produits provoquant des allergies ou intolérances"
+msgid "Allergens"
+msgstr "Allergènes"
```

Comment:
FYI there is a attribute_group_allergens_name (just in case you wanted to copy translation in all languages)